NativeApp/src/components/Home/CustomDay/index.js

```diff
@@ -28,19 +36,29 @@ const CustomDay = (props) => {
     );
   };
 
+  const renderFiller = (date) => {
+    let filler;
+    if (!sameDate) {
+      if (date || (!startingDate && !endingDate)) {
```

Comment:
is there a need for 2 separate if statements? - not a huge deal, just might be cleaner as one

Reply:
It was really for my own benefit, easier to read while I was working on it. But can change to a one liner. :)

Reply:
@kirsty-unosquare I think it's easier to read this way as well.